besu/src/test/java/org/hyperledger/besu/cli/PublicKeySubCommandTest.java

```diff
@@ -141,6 +177,30 @@ public void callingPublicKeyExportSubCommandWithFilePathMustWritePublicKeyInThis
     assertThat(commandErrorOutput.toString()).isEmpty();
   }
 
+  @Test
+  public void callingPublicKeyExportSubCommandWithPrivateKeyFileMustWriteKeyToStandardOutput()
```

Comment:
What is the behavior if the file doesn't exist or cannot be read?

Reply:
It throws an exception. I just added a test that reflects that behavior